config/environments/production.rb

```diff
@@ -59,7 +59,7 @@
   config.log_level = ENV.fetch("RAILS_LOG_LEVEL", "info")
 
   # Use a different cache store in production.
-  # config.cache_store = :mem_cache_store
+  config.cache_store = :redis_cache_store, { url: ENV["REDIS_URL"] }
```

Comment:
Heads up — this'll need to be configured before deploying.

We're already using `Rails.cache.fetch`. But it's not doing anything because no cache store is configured. This'll fix that.

Reply:
Thanks for the heads up.  cc @Shpigford, we switched this over to Hatchbox recently correct?